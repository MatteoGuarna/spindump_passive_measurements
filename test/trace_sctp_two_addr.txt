A short test where two SCTP associations are established towards two different addresses.
