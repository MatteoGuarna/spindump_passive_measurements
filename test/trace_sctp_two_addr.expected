SCTP 10.0.2.5 <-> 10.0.2.5 3455794611:0 (33363:62324) at 1581329892995723 new starting packets 1 0 bytes 68 0 bandwidth 68 0
SCTP 127.0.0.1 <-> 127.0.0.1 2290022628:0 (52542:62324) at 1581329896033725 new starting packets 1 0 bytes 84 0 bandwidth 84 0
SCTP 10.0.2.5 <-> 10.0.2.5 3455794611:2370241070 (33363:62324) at 1581329897996785 measurement up right 19 packets 3 2 bytes 396 328 bandwidth 332 328
SCTP 10.0.2.5 <-> 10.0.2.5 3455794611:2370241070 (33363:62324) at 1581329897996867 measurement up left 8 packets 3 4 bytes 396 440 bandwidth 332 328
SCTP 127.0.0.1 <-> 127.0.0.1 2290022628:2870338105 (52542:62324) at 1581329901034172 measurement up right 18 packets 3 2 bytes 444 376 bandwidth 380 376
SCTP 127.0.0.1 <-> 127.0.0.1 2290022628:2870338105 (52542:62324) at 1581329901034238 measurement up left 7 packets 3 4 bytes 444 488 bandwidth 380 376
